swri_profiler_tools/src/profile.cpp

```diff
@@ -300,7 +300,7 @@ void Profile::rebuildFlatIndex()
 }
 
 // Compares the first N items of two string lists.
-static bool compareInitialStringList(
+bool compareInitialStringList(
```

Comment:
I did this so long ago that I don't actually remember, but I suspect it was related to this warning:

```
Warnings   << swri_profiler_tools:make logs/swri_profiler_tools/build.make.000.log                                                                                                                           
src/swri_profiler/swri_profiler_tools/src/profile.cpp:303:13: warning: ‘bool swri_profiler_tools::compareInitialStringList(const QStringList&, const QStringList&)’ defined but not used [-Wunused-function]
 static bool compareInitialStringList(
             ^
cd build/swri_profiler_tools; catkin build --get-env swri_profiler_tools | catkin env -si  /usr/bin/make --jobserver-fds=6,7 -j; cd 
```

It seems that the actual problem here is that the function is never used (in this file or elsewhere). Is this a half-implemented feature, or should it be removed?